UPGRADING.md

```diff
@@ -10,6 +10,18 @@ gem 'grape-swagger'
 gem 'grape-swagger-entity'
 ```
 
+`add_swagger_documentation` has changed from
+``` ruby
+  add_swagger_documentation \
+    api_version: 'v1'
+```
+to
+
+``` ruby
+  add_swagger_documentation \
+    version: 'v1'
```

Comment:
This is what i'm currently using in my project and if that's the case it looks like it forwards it on.